.gitignore

```diff
@@ -1,2 +1,4 @@
 dist
 vendor
+tmp
+sopstool
```

Comment:
nice - good to add this to prevent that bin from actually getting in there. make sure to squash & merge to kill that outta raw git history